yolapy/resources/user.py

```diff
@@ -86,6 +88,18 @@ def get_sso_open_site_url(self, user_id, site_id=None, locale=None):
                 'locale': locale
             }).json()['url']
 
+    def get_sso_url(self, user_id, site_id=None,
```

Comment:
How do we get sso url for old users?

Reply:
https://github.com/yola/yolapy/blob/0ea9c7a49d1146bb2a4a01ec491a465fcbb82be4/yolapy/resources/user.py#L70-L89

Reply:
I would not add a separate method for this. It looks like it rather belongs to `get_open_site_url` with `if self.is_ws` inside.

Reply:
It's service, not model. We don't store `is_ws` there. And I don't think that it's good idea to add additional argument `is_ws`, because for `get_sso_url` we have additional option `destination`

Reply:
So you're going to have `if is ws` somewhere?

Reply:
yes, in bottomline